hooks/basic/publish.py

```diff
@@ -252,38 +252,47 @@ def validate(self, task_settings, item):
                 }
             )
 
-        # ---- ensure the published file(s) don't already exist on disk
-
-        conflict_info = None
         if item.properties.get("is_sequence"):
-            seq_pattern = publisher.util.get_path_for_frame(item.properties.publish_path, "*")
-            seq_files = [f for f in glob.iglob(seq_pattern) if os.path.isfile(f)]
-
-            if seq_files:
-                conflict_info = (
-                    "The following files already exist!<br>"
-                    "<pre>%s</pre>" % (pprint.pformat(seq_files),)
-                )
+            path = publisher.util.get_path_for_frame(item.properties.path, "*")
+            publish_path = publisher.util.get_path_for_frame(item.properties.publish_path, "*")
```

Comment:
Please suffix these with `_pattern` or `_glob` to distinguish them from the actual resolvable paths.